app/services/state_file/state_information_service.rb

```diff
@@ -0,0 +1,43 @@
+module StateFile
+  class StateInformationService
+    class << self
+      def active_states # TODO thinking this could also be called `active_state_codes` or just `state_codes`?
+        STATE_INFO.keys.map(&:to_s)
+      end
+
+      def state_name(state_code)
+        unless active_states.include?(state_code)
+          raise StandardError, state_code
+        end
+
+        States.name_for_key(state_code.upcase)
+      end
+
+      def state_code_to_name_map
```

Comment:
I think this would be simpler as `active_states.to_h { |state_code, _| [state_code, state_name(state_code) }` but I haven't tested that

Reply:
done!